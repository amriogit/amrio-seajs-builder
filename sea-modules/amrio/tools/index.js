define("amrio/tools/index",[],function(e,o,t){"use strict";t.exports={log:function(){console.log(arguments)}}});

