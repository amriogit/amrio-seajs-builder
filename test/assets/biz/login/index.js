define(function(require, exports, module) {
    'use strict'

    var $ = require('$')
    var angular = require('angular')
    require('amrio/tips/style.css')
    var tips = require('amrio/tips/index')
    require('amrio/tools/index')

    var msgs = require('./error-msg')
    require('./style.css')

    module.exports = function() {
        tips.show('login success')
    }
})