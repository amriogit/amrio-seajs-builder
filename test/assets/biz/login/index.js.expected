define("biz/login/index", [ "$", "angular", "bootstrap" ], function(require, exports, module) {
    "use strict";
    var $ = require("$");
    var angular = require("angular");
    require("amrio/tips/style.css");
    var tips = require("amrio/tips/index");
    require("amrio/tools/index");
    var msgs = require("./error-msg");
    require("./style.css");
    module.exports = function() {
        tips.show("login success");
    };
});

define("amrio/tips/style.css", [], function() {
    seajs.importStyle('@import url("./base.css"); .ui-tips{ font-size: 12px; border: 1px #f00 solid; } .ui-tips-content{ background-color: #fff; }');
});

define("amrio/tips/index", [], function(require, exports, module) {
    "use strict";
    var $ = require("$");
    var bootstrap = require("bootstrap");
    require("./style.css");
    require("./helper");
    var tools = require("amrio/tools/index");
    module.exports = {
        show: function(text) {
            text && tools.log(text);
        }
    };
});

undefined;

define("amrio/tips/helper", [], function(require, exports, module) {
    "use strict";
    module.exports = {
        helper: {}
    };
});

define("amrio/tools/index", [], function(require, exports, module) {
    "use strict";
    module.exports = {
        log: function() {
            console.log(arguments);
        }
    };
});

define("biz/login/error-msg", [], function(require, exports, module) {
    "use strict";
    require("amrio/tools/index");
    module.exports = {
        error: "error"
    };
});

undefined;

define("biz/login/style.css", [], function() {
    seajs.importStyle(".ui-tips{ color: #eee; border: 1px #666 solid; }");
});